mui-org/material-ui
# [docs] Reword unsupported material components notes

# The issue

As mentioned in #16513, some visitors to the docs might read the alerts about "no longer documented" components and think that MUI will stop supporting those components. However, the alert only means to warn users that the [Material Design specifcation](https://material.io/) is no longer supporting the components in question.

# The solution

Update the wording to be more clear that MUI will continue to support these components:
- Steppers
- Expansion Panel
- Text Field (without variant)

# Visuals

Area | Before | After
----- | ----- | -----
Steppers | ![image](https://user-images.githubusercontent.com/11624407/61574441-79106280-aa85-11e9-8bd6-baf8d1cd5752.png) | ![image](https://user-images.githubusercontent.com/11624407/61579502-79cbe780-aacb-11e9-9696-9071fcaf8ccc.png)
Expansion Panels | ![image](https://user-images.githubusercontent.com/11624407/61574452-a0672f80-aa85-11e9-9aca-35b456031050.png) | ![image](https://user-images.githubusercontent.com/11624407/61579504-83ede600-aacb-11e9-8135-d55567a0d81c.png)
Text Field (without variant) | ![image](https://user-images.githubusercontent.com/11624407/61574462-bbd23a80-aa85-11e9-9405-0332304ec042.png) | ![image](https://user-images.githubusercontent.com/11624407/61579511-8ea87b00-aacb-11e9-9e55-aca65d970dc9.png)

# To test

- Get this code.
- Ensure that you see alerts for the changed areas as expected.
